### ISSUE TITLE
Wrong syntax in the README

### ISSUE THREAD

**Issue by okalm** [Xanthus58/FileSorterX#16]
Hello,

While trying out your software I just followed the example given in the README and it seems the syntax isn't correct.

```
/tmp/00 % ~/bin/FileSorterX_v1.2.0_x86_64-unknown-freebsd customsort --input /tmp/00 --output /tmp/00/textfolder --extension txt --verbose 
error: unexpected argument '--input' found

  tip: a similar argument exists: '--inputdir'

Usage: FileSorterX_v1.2.0_x86_64-unknown-freebsd customsort <--inputdir <INPUTDIR>|--outputdir <OUTPUTDIR>|--extension <EXTENSION>|--verbose|--log>

For more information, try '--help'.
```

Then I just used the syntax given by the tip and `usage` and it worked.

```
/tmp/00 % ~/bin/FileSorterX_v1.2.0_x86_64-unknown-freebsd customsort --inputdir /tmp/00 --outputdir /tmp/00/textfolder --extension txt --verbose                                     % 16:05:02
Moved file: "/tmp/00/file_3.txt" to "/tmp/00/textfolder"
Moved file: "/tmp/00/file_4.txt" to "/tmp/00/textfolder"
Moved file: "/tmp/00/file_5.txt" to "/tmp/00/textfolder"
Moved file: "/tmp/00/file_2.txt" to "/tmp/00/textfolder"
Moved file: "/tmp/00/file_1.txt" to "/tmp/00/textfolder"
Moved file: "/tmp/00/file_6.txt" to "/tmp/00/textfolder"
```

So `--inputdir` `--outputdir` are correct but not  `--input` and `--output` .

OS: FreeBSD 13.1-RELEASE
Package version: FileSorterX_v1.2.0_x86_64-unknown-freebsd

**Comment by Xanthus58** [comment]
Hey I'm currently aware of this issue; I'm waiting for our documentation creator @WolfenXVII to finish making the updated readme. It'll include various different updates and changes.

It will be released in version v1.2.1 Which is entirely aimed to be a bug fix and documentation overhaul patch primarily. You can see the status of v1.2.1 here https://github.com/Xanthus58/FileSorterX/milestone/2

**Comment by Xanthus58** [comment]
Fixed in the new readme
